### ISSUE TITLE
Unknown bottom blob 'label' (layer 'loss1/loss', bottom index 1)

### ISSUE THREAD

**Issue by shinebay** [BestiVictory/ILGnet#2]
新手试运行了代码：
import numpy as np
import matplotlib.pyplot as plt
caffe_root = '/opt/caffe/'
import sys
sys.path.insert(0, caffe_root + 'python')
import caffe
MODEL_FILE = caffe_root + 'ILGnet/deploy.prototxt'
PRETRAINED = caffe_root + 'ILGnet/ILGnet-AVA2.caffemodel'
IMAGE_FILE = caffe_root+'examples/images/cat.jpg'
mean_file=caffe_root + 'ILGnet/AVA2_mean.npy'
caffe.set_mode_cpu()
net = caffe.Classifier(MODEL_FILE, PRETRAINED,
mean=np.load(mean_file).mean(1).mean(1),
channel_swap=(2,1,0),
raw_scale=255,
image_dims=(227, 227))
input_image = caffe.io.load_image(IMAGE_FILE)
plt.imshow(input_image)
prediction = net.predict([input_image])
plt.plot(prediction[0])
plt.show()
print 'predicted class:', prediction[0].argmax()
不知道哪里有错，希望能解答

**Comment by BestiVictory** [comment]
你可以将deploy.prototxt里面的最后一层的类型SOFTMAX_LOSS改为SOFTMAX

**Comment by shinebay** [comment]
@BestiVictory 谢谢，按照您的指示：deploy.prototxt里面的最后一层的类型
layers {
  name: "loss1/loss"
  type: SOFTMAX
  bottom: "loss1/classifier_cjy"
  bottom: "label"
}
更改后运行：
F0502 11:37:13.672961    60 insert_splits.cpp:35] Unknown bottom blob 'label' (layer 'loss1/loss', bottom index 1)
*** Check failure stack trace: ***
Aborted (core dumped)
还是这个错误，麻烦您了

**Comment by BestiVictory** [comment]
你的代码用到了net.predict函数，需要把最后一层改成这样：
layers {
name: "loss1/loss"
type: SOFTMAX
bottom: "loss1/classifier_cjy"
top: "prob"
}

**Comment by shinebay** [comment]
@BestiVictory 额。。。改了之后就是：
F0503 09:54:48.940515    67 insert_splits.cpp:35] Unknown bottom blob 'prob' (layer 'loss1/loss', bottom index 1)

**Comment by BestiVictory** [comment]
抱歉由于工作原因没有及时回复你的消息，我已经更新了deploy及test的脚本，你可以使用我的脚本及deploy文件进行测试。
